hpat/tests_perf/test_perf_utils.py

```diff
@@ -72,24 +90,28 @@ def perf_data_gen(tmpl, max_item_len, max_bytes_size):
 
                 return: list of strings
     """
-
     result = []
-    obj_size = get_size(tmpl)
+    while get_size(tmpl) < max_bytes_size:
+        result.extend(multiply_data(tmpl, max_item_len))
 
-    while (obj_size < max_bytes_size) and (obj_size >= 0):
-        for item in tmpl:
-            local_item = item
-            local_item_len = len(local_item)
+    return result
 
-            while (local_item_len < max_item_len) and (local_item_len >= 0):
-                local_item += item
-                local_item_len = len(local_item)
 
-            result.append(local_item)
+def perf_data_gen_fixed_len(tmpl, max_item_len, max_obj_len):
```

Comment:
Code duplication. No need to create second function for the first one with one line difference.